.github/scripts/get_sunset_apis.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# get_sunset_apis.sh - Identifies API versions scheduled for sunset
+#
+# This script queries an OpenAPI specification to find API versions that will be
+# sunset within a specified date range. When sunset APIs are found,
+# it generates a hash code from the results to help with deduplication in
+# downstream processes (like JIRA ticket creation) and saves the list to a file.
+#
+# Usage:
+#   ./get_sunset_apis.sh <openapi_spec_url> <to_date>
+#
+# Parameters:
+#   openapi_spec_url - URL of the OpenAPI specification to analyze
+#   to_date - End date for the sunset check (format: YY-MM-DD)
+#
+# Outputs:
+#   - Prints information about sunset APIs to stdout
+#   - Writes hash_code_sunset_apis to GitHub Actions output if sunset APIs are found
+#   - Creates sunset_apis.json containing the list of sunset APIs if any are found
+#
+# Dependencies:
+#   - foascli - CLI tool for querying sunset information from OpenAPI specs
+#   - jq - JSON processor for hash code generation
+#
+# Exit Status:
+#   - 0 if the script completes successfully, regardless if sunset APIs are found
+#   - Non-zero if any commands fail (due to set -e)
+set -eou pipefail
+
+openapi_spec_url="$1"
```

Comment:
are these mandatory? should we add validation?

Reply:
updated

Reply:
@fmenezes Ready for review 🙏